Halt api (#1749)
MetaMask/metamask-docs: services/reference/_partials/error-codes.mdx

> Here you can find the JSON-RPC API endpoints. You [can call these APIs using a variety of tools](../quickstart).

Reviewer: ```suggestion
This section contains the JSON-RPC API endpoints on the left sidebar. You can [call these APIs using a variety of tools](../quickstart).
```

## Error codes

You may encounter various errors when interacting with a network:

> - [**JSON-RPC errors**](#json-rpc-errors)
>   operations on the blockchain network. It means the server has received the JSON-RPC

Reviewer: ```suggestion
- [**JSON-RPC errors**](#json-rpc-errors):
  These are related to operations on the blockchain network. It means the server has received the JSON-RPC
```

  request but encountered an issue processing it. Causes might include invalid parameters, a method not
  found, or execution errors related to the requested operation.

- [**HTTP errors**](#http-errors): These happen at the transport layer during data transmission to the blockchain
  network. They could stem from Infura-related issues like rate limits, API key problems, or
  service availability issues.

- [**Smart contract errors**](#smart-contract-errors): These arise during attempts to execute transactions in the EVM
  involving smart contracts.

### JSON-RPC errors

> Error codes returned by Infura’s RPC service network APIs can vary slightly between implementations, but

Reviewer: ```suggestion
Error codes returned by Infura's RPC service network APIs can vary slightly between implementations, but
```

they follow the [JSON-RPC 2.0 specification](https://www.jsonrpc.org/specification#error_object)
and Ethereum-specific conventions.

> Below is a table listing the error codes, their messages, and meanings. The "Standard" category includes

Reviewer: ```suggestion
The following tables lists the error codes, their messages, and meanings. The "Standard" category includes
```

> The following tables lists the error codes, their messages, and meanings. The "Standard" category includes

Reviewer: ```suggestion
The following table lists the error codes, their messages, and meanings. The "Standard" category includes
```
